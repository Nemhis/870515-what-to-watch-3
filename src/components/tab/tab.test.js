import React from 'react';
import renderer from 'react-test-renderer';

import Tab from './tab.jsx';

it(`Render correctly <Tabs>`, () => {
  const tree = renderer
    .create(<Tab name={`test tab`}><span>Tab children</span></Tab>).toJSON();

  expect(tree).toMatchSnapshot();
});

