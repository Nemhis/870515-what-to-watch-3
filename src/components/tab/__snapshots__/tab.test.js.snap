// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render correctly <Tabs> 1`] = `
<span>
  Tab children
</span>
`;
